admin/invoice.php

```diff
@@ -6,16 +6,12 @@
  * @version $Id: DrByte 2020 May 20 Modified in v1.5.7 $
  */
 require('includes/application_top.php');
-// To override the $show_* values, see 
+// To override the $show_* or $attr_img_width values, see 
 // https://docs.zen-cart.com/user/admin/site_specific_overrides/
 if (!isset($show_product_images)) $show_product_images = true;
 if (!isset($show_attrib_images)) $show_attrib_images = true;
 $img_width = defined('IMAGE_ON_INVOICE_IMAGE_WIDTH') ? (int)IMAGE_ON_INVOICE_IMAGE_WIDTH : '100';
-$attr_img_width = '25';
-/*
- * Notifier to allow defaults to be changed
- */
-$zco_notifier->notify('NOTIFY_ADMIN_INVOICE_DEFAULTS', '', $show_product_images, $show_attrib_images, $attr_img_width);
+if (!isset($attr_img_width)) $attr_img_width = '25';
```

Comment:
https://www.php-fig.org/psr/psr-12/#51-if-elseif-else

Reply:
Does that mean I should change 
lines 11-12 as well to 
if (condition) {
    action;
}
as well I copied the format from these lines. Happy to make change, just trying to be consistent.

Same in packingslip,php

Reply:
Have changed them all. If its wrong, I'll change the old ones back.